orchagent/lagids.lua

```diff
@@ -0,0 +1,90 @@
+-- KEYS - None
+-- ARGV[1] - operation (add/del/get)
+-- ARGV[2] - lag name
+-- ARGV[3] - current lag id (for "add" operation only)
+
+-- return lagid if success for "add"/"del"
+-- return 0 if lag not exists for "del"
+-- return -1 if lag table full for "add"
+-- return -2 if log does exist for "get"
```

Comment:
Thanks for these comments, misspelt "lag" as "log"

Reply:
Sorry for the typo. I'll fix them.

Reply:
Fixed